dotcom-rendering/src/web/components/Callout/CalloutComponents.tsx

```diff
@@ -91,21 +98,38 @@ const supportingText = css`
 	color: ${neutral[46]};
 `;
 
-interface Props {
+export const CalloutShare = ({
+	format,
+}: {
 	format: ArticleFormat;
-}
-
-export const CalloutShare = ({ format }: Props) => {
+	title: string;
+}) => {
 	const [isCopied, setIsCopied] = useState(false);
 
 	const onShare = async () => {
 		const url = window.location.href;
+		if ('share' in navigator) {
+			const shareTitle = `
+			Share your experience: ${'PLACEHOLDER TITLE'}
+			`;
+			const shareText = `
+			I saw this callout on an article I was reading and thought you might like to share your story.
+			${url}
+			You can share your story by using the form on this article, or by contacting us on WhatsApp or Telegram.
+					`;
+			await navigator.share({
+				title: shareTitle,
+				text: shareText,
+			});
+		}
+
 		if ('clipboard' in navigator) {
 			await navigator.clipboard.writeText(url);
```

Comment:
I can message Ben to ask for clarification on how it should behave on the web?

Reply:
Since speaking to Bella, I think this is correct - if the browser can't invoke the native share functionality, it just copies the url to the clipboard for the user to manually share.